finder-app/Makefile

```diff
@@ -0,0 +1,14 @@
+ifdef CROSS_COMPILE
```

Comment:
@parthkharade see assignment videos/slides for some more simple logic for this step.